include/UIKit/UIView.h

```diff
@@ -379,11 +379,10 @@ UIKIT_EXPORT_CLASS
 @end
 
 // WinObjC Xaml Extensions
-@class WXFrameworkElement;
 @interface UIView (UIKitXamlExtensions)
-+ (WXFrameworkElement*)createXamlElement;
-@property (nonatomic, readonly, strong) WXFrameworkElement* xamlElement;
-- (instancetype)initWithFrame:(CGRect)frame xamlElement:(WXFrameworkElement*)xamlElement;
++ (RTObject*)createXamlElement;
+@property (nonatomic, readonly, strong) RTObject* xamlElement;
```

Comment:
>RTObject* [](start = 40, length = 9)

This is a breaking change for our clients. Not only has the type of this property changed, but you can't downcast it to WXFrameworkElement* or WXCTextField* or whatever, because the object returned is genuinely not one of those things. You also can't just go ahead and invoke WXFrameworkElement methods on the object. You now have to use rt_dynamic_cast to do a downcast first.